e2e/rook-test-framework/rook-infra-manager/scripts/rook-dind-cluster-v1.5.sh

```diff
@@ -0,0 +1,1042 @@
+#!/bin/bash
```

Comment:
it turned out be very useful when I moved to v1.6 testing, because the v1.5 scenarios were working but not the v1.6... I think i am going to leave it in for testing of regressions of rook between 1.5 and 1.6 for now but not make it available to run as a part of CI. Is this acceptable